clayui.com/src/components/Editor/index.js

```diff
@@ -84,19 +88,31 @@ const Editor = ({
 				/>
 
 				<div style={{padding: '10px'}}>
-					<ClayButtonWithIcon
-						className="btn-copy"
-						displayType="unstyled"
-						small
-						spritemap={spritemap}
-						symbol={'paste'}
-						title={'Copy'}
-					/>
+					{snippets.length === 1 && collapseCode && (
```

Comment:
Should this be `!!snippets.length` meaning that only show if snippets exist? Or is this explicitly checking against a single snippet

Reply:
It's supposed to show the `Code Sample (expand to see it)` text only when there is a single snippet of code - when no tabs are displayed, primarily in the HTML/CSS examples, because they don't have language variants.